lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -469,6 +469,8 @@ impl Dom {
 pub struct DomLayers {
     /// Back DOM scene layer.
     pub back:           DomScene,
+    /// Back DOM scene layer with disabled panning.
+    pub panel:          DomScene,
```

Comment:
This name is not semantically explanable. If you go to someone from another team and ask "What is the responsibility of dom layer 'panel'" - would they answer it correctly?

Reply:
> This name is not semantically explanable. If you go to someone from another team and ask "What is the responsibility of dom layer 'panel'" - would they answer it correctly?

I'd answer: to keep panels, I mean: GUI components being a fixed panel. Is it the responsibility of this layer indeed?

Reply:
As a stated in the PR description, the naming is hard here and I would love to hear alternative proposals.
The most obvious name would be `welcome_screen` - as this layer is used by welcome screen only at the moment. But this can easily change in future and we'll be forced to rename it.
Second obvious name is probably smth like `back_without_panning`, as `back_with_fixed_position` doesn't work. However, this naming is subjectively too long and doesn't show a connection with `HardcodedLayers::panel`. It is not clear why the first one is called `without_panning`, and the later one is just `panel`.

Reply:
To clarify - this is potentially useful feature that would be used by other components, but for now we can call it `welcome_screen` and create another one for other components later.

Reply:
I'm all in for `welcome_screen` now, as `panel` can be anything and we did not establish this naming anywhere in the codebase. As I explained in other places in the PR, `back_` is a bad name because this should be in front, not in the back!

Reply:
Renamed to `welcome_screen`